app/Console/Commands/TagPosts.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Tag;
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+
+class TagPosts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:tagposts {--tag=: The id of the tag to use.} {--posts=: Comma-separated list of the post_ids to update.}';
```

Comment:
Don't want to overcomplicate this, but in other commands where do things like that we usually read in things like this from a CSV (but that in itself is kind of a pain because it has to get uploaded to AWS). Wondering how many post IDs will be passed here and if you think passing them in through a CSV is more trouble than it's worth?

Reply:
Yea I think that could make sense for larrrrge batches of updates but that doesn't mean that it is not useful to have an easy command to reach to when we need to update a set of post. Also, this is really something that can just be done directly in the DB but having an easy command to reach to seems like a win. There has only been one use case for this really so far, but I can see this being useful in the future. Doesn't really bother me if is a large set of posts but yea I can see that being cumbersome. Something to keep in mind if this becomes a frequent request!

Reply:
@sbsmith86 awesome! makes total sense to me to have a quick way to add these tags, I'm on board 🚢

Reply:
yea hmmm. let me make a `@TODO` to send these quasar. I rather us continue using the service-repository pattern so we can really see what all the business logic is around events in rogue, even though it requires the mental lift to remember to do it.

Reply:
@sbsmith86 sounds good to me!

Reply:
@katiecrane updated!